### DISCUSSION TITLE
CT2 as Long Term Statistics

### DISCUSSION THREAD

**Discussion by sofapi** [nathanmarlor/foxess_em#330] [Q&A]
Is it possible to have CT2 Sensor as a Long Term Statistics entity?   (Sensor.CT2_Meter)

I have connected CT2 to the H1 Inverter and it produces solar data but I cannot configure this in "Solar Panels" within the "Energy Dashboard".

**Comment by mattwilson1066** [comment]
I'd be interested to know this too. I also have an Myenergi Eddi and Harvi, so I am using one of those CT's to log the solar data from my F4600 inverter, but it would be nicer if I can set the CT2 to be the main solar in the Energy Dashboard.
